ssm/ssm2.py

```diff
@@ -15,6 +15,8 @@
 """
 from __future__ import print_function
 
+import pkg_resources
```

Comment:
We need to be careful about adding extra dependencies. `pkg_resources` seems to be advised against and once you're on Python 3 `importlib.metadata` seems to be the way to go: https://docs.python.org/3/library/importlib.metadata.html#distribution-versions

So we could add a version check in a `try...except` that'll skip it if we're on Python2.

Reply:
Oh, the import will need to be in a `try...except` for `ImportError`

Reply:
Alternative solution to making `pkg_resources` optional added.